sqs/provider.go

```diff
@@ -90,6 +103,13 @@ func (s *Provider) Provision(ctx context.Context, provisionData provideriface.Pr
 	}, nil
 }
 
+func mkParameter(name string, value int) *cloudformation.Parameter {
+	return &cloudformation.Parameter{
+		ParameterKey:   aws.String(name),
+		ParameterValue: aws.String(fmt.Sprint(value)),
```

Comment:
style: could we use `strconv.Itoa` or a `"%d"` formatter here if this is intended to always be an `int` (go vet and tools can enforce types for the format strings, but can't help here)... or change the mkParamater argument to be an interface if it is intended to be a generic stringify-anything-ifier? .... looks like it's in a middleground at the moment.